### DISCUSSION TITLE
ScrollView is causing TabView to disappear

### DISCUSSION THREAD

**Discussion by Zein0** [react-native-community/react-native-tab-view#1363] [Q&A]
I tried writing this code my scene is disapearing until I add it manually I don't want to measure it unless I have to,
code:
`` <ScrollView style={{styles.container}}>
    <TabView
      navigationState={{ index, routes }}
      renderScene={renderScene}
      onIndexChange={setIndex}
      initialLayout={initialLayout}
      style={styles.tabs}
    />
    </ScrollView>``
I want my whole page to be scrollable how can this be done?

**Comment by ElliotWood13** [comment]
DId you get round this @Zein0? Just wondered if you had a solution

**Reply by Zein0** [reply]
@ElliotWood13 Hello I end up adding height*x where height relative to the screen height to the TabView no issues with UI has happened with different resolutions

**Reply by ElliotWood13** [reply]
I'll give it a go, thanks a lot for responding @Zein0!
